src/data/polynomial/algebra_map.lean

```diff
@@ -215,6 +215,9 @@ alg_hom_ext $ by simp only [aeval_X, alg_hom.comp_apply]
 @[simp] theorem aeval_X_left : aeval (X : R[X]) = alg_hom.id R R[X] :=
 alg_hom_ext $ aeval_X X
 
+theorem aeval_X_left_apply (p : R[X]) : aeval X p = p :=
```

Comment:
I guess this cannot be `simp` since `simp` can already prove it, right? I mean, the linter would complain.

Reply:
Yes.